lib/resources/test/cypress/integration/app.e2e.js

```diff
@@ -0,0 +1,13 @@
+describe('Aurelia skeleton app', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should load the page and display the initial page title', () => {
+    cy.title().should('equal', 'Aurelia Navigation Skeleton');
```

Comment:
A small problem. There is newly added scaffold.
```
Which features do you want scaffolded into your project?

1. None
   Just give me an empty project.
2. Navigation
   Add a router and some sample routes. (Also adds Bootstrap 3 and Font Awesome.)
```

When user picks "navigation" scaffold, the e2e needs to be different. Again, you can look into existing protractor setup.

Reply:
Ok will do 👍